packages/bpk-component-accordion/src/BpkAccordionItem.scss

```diff
@@ -32,18 +32,18 @@
     cursor: pointer;
     appearance: none;
 
-    @include bpk-themeable-property(color, --bpk-accordion-color, $bpk-link-color);
+    @include bpk-themeable-property(color, --bpk-accordion-color, $bpk-color-sky-blue-shade-03);
 
     @include bpk-rtl {
       text-align: right;
     }
 
     @include bpk-hover {
-      @include bpk-themeable-property(color, --bpk-accordion-hover-color, $bpk-link-hover-color);
+      @include bpk-themeable-property(color, --bpk-accordion-hover-color, $bpk-color-sky-blue-shade-03);
```

Comment:
This has changed due to link beinging used in multiple places and in the new brand is not the same colour